blocks/feeds-source-collections-block/sources/collections.js

```diff
@@ -51,8 +70,22 @@ const fetch = async (key = {}) => {
     ...(contentAlias && { content_alias: contentAlias }),
   }
 
-  // ids results does not include content_elements unless specified in included_fields
-  const idsIncludedFields = includedFields || defaultIncludedFields
+  // limit C-API response to just this websites sections to reduce size
+  ansFields.push(`websites.${site}`)
```

Comment:
If `ansFields` is a global variable when it's deployed, then this could create a periodic and difficult-to-debug problem on AWS Lambda. 

AWS Lambda sometimes reuses their internal VMs; so modifying a global variable can cause one user's request to affect the next user's request. Since we have little control over how AWS spins down these internal VMs, this can propagate errors that are difficult to track down and fix.